pyproject.toml

```diff
@@ -28,8 +28,11 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-nautobot = "^1.0.3"
+nautobot = "^1.1.4"
 diffsync = "^1.3.0"
+# Added pinned typed-add to overcome issues with black version
+typed-ast = "^1.5.0"
+markdown = "<=3.3.4"
```

Comment:
Why these additions? If typed-ast is a Black dependency, it should be a dev-dependency not an main dependency. Markdown is a Nautobot dependency so this plugin absolutely should not be specifying its own version dependency on it.